app/views/coronavirus_local_restrictions/_christmas_rules.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render "govuk_publishing_components/components/heading", {
+    text: t("coronavirus_local_restrictions.results.christmas_rules.heading"),
+    font_size: "m",
+    margin_bottom: 4,
+    mobile_top_margin: true,
+  } %>
+
+<p class="govuk-body">
+  <%= t("coronavirus_local_restrictions.results.christmas_rules.body") %>
+</p>
+
+<a href="<%= t("coronavirus_local_restrictions.results.christmas_rules.guidance.link") %>"
+  class="govuk-button govuk-button--secondary govuk-!-margin-bottom-8" role="button" data-module="govuk-button">
```

Comment:
What's the purpose of the data-module ?

Reply:
Good spot - should have removed - copy and paste error from https://design-system.service.gov.uk/components/button/#secondary-buttons